docker-compose.yaml

```diff
@@ -44,6 +50,36 @@ services:
             - ./backend/supervisor.d/:/etc/supervisor/conf.d/
             # uncomment after Step 2 of README
             #- ./local_configs/backend/supervisor.d/:/etc/supervisor/conf.d/
+    monitor:
+        image: inspiregroup/artemis-monitor:${SYSTEM_VERSION}
+        container_name: monitor
+        depends_on:
+            - rabbitmq
+        networks:
+            - artemis
+        expose:
+            - ${MON_SUPERVISOR_PORT}
+        environment:
+            RABBITMQ_USER: ${RABBITMQ_USER}
+            RABBITMQ_PASS: ${RABBITMQ_PASS}
+            RABBITMQ_HOST: ${RABBITMQ_HOST}
+            RABBITMQ_PORT: ${RABBITMQ_PORT}
+            RIS_ID: ${RIS_ID}
+            MON_SUPERVISOR_HOST: ${MON_SUPERVISOR_HOST}
+            MON_SUPERVISOR_PORT: ${MON_SUPERVISOR_PORT}
+            REDIS_HOST: ${REDIS_HOST}
+            REDIS_PORT: ${REDIS_PORT}
+        volumes:
+            # uncomment to run from source code (only if you build from source)
+            # - ./monitor/:/root/
+            # comment after Step 2 of README
+            - ./monitor/configs/:/etc/artemis/
+            # uncomment after Step 2 of README
+            # - ./local_configs/backend/:/etc/artemis/
```

Comment:
@slowr after merging, we need to also check how Step 3 of https://github.com/FORTH-ICS-INSPIRE/artemis/wiki#setup-tool should be revised to have separate backend and monitor configurations. The supervisor conf for monitor does not need to be there I think (will confuse the current deployers) since they will also not need to change it anyway.

Reply:
noting this here so that we do not forget:
1) for new users the following commands should be OK:
```
mkdir -p local_configs && \
mkdir -p local_configs/backend && \
mkdir -p local_configs/monitor && \
mkdir -p local_configs/frontend && \
cp -rn backend/configs/* local_configs/backend && \
cp -rn backend/supervisor.d local_configs/backend && \
cp -rn monitor/configs/* local_configs/monitor && \
cp -rn monitor/supervisor.d local_configs/monitor && \
cp -rn frontend/webapp/configs/* local_configs/frontend
```
2) for old users the following needs to be done:
```
mkdir -p local_configs/monitor && \
cp -rn monitor/configs/* local_configs/monitor && \
cp -rn monitor/supervisor.d local_configs/monitor && \
if [ -e local_configs/backend/exabgp.conf ]; then mv local_configs/backend/exabgp.conf local_configs/monitor/; fi && \
if [ -e local_configs/backend/logging.yaml ]; then cp local_configs/backend/logging.yaml local_configs/monitor/; fi &&
```